lib/dfTools.class.php

```diff
@@ -1376,4 +1376,65 @@ public static function preg_replace($pattern, $replacement, $subject)
     {
         return is_null($subject) ? null : preg_replace($pattern, $replacement, $subject);
     }
+
+    public static function get_min_variant_prices($products, $include_taxes)
+    {
+        $min_prices_by_product_id = [];
+        foreach ($products as $product) {
+            if (self::is_parent($product)) {
+                continue;
+            }
+
+            $product_id = $product['id_product'];
+            $variant_id = $product['id_product_attribute'];
+            $variant_price = self::get_price($product_id, $include_taxes, $variant_id);
+            $variant_onsale_price = self::get_onsale_price($product_id, $include_taxes, $variant_id);
+
+            if (key_exists($product_id, $min_prices_by_product_id)) {
+                $current_min_prices = $min_prices_by_product_id[$product_id];
+
+                // Even though, in order to track the minimum, we can only focus on
+                // the sales price, we still need both prices of the variant
+                // in order to properly and consistenly populate the price of
+                // the parent to show the proper price vs sale_price when searching
+                // in the layer
```

Comment:
I would replace it with a multiline comment :smile: 

```suggestion
                 /*
                Even though, in order to track the minimum, we can only focus on
                the sales price, we still need both prices of the variant
                in order to properly and consistenly populate the price of
                the parent to show the proper price vs sale_price when searching
                in the layer
                */
```

Reply:
Hecho! Gracias por la sugerencia David!